.github/workflows/prune-images.yml

```diff
@@ -62,4 +67,11 @@ jobs:
               --apps="${APPLICATIONS}" \
               --prefix=${PREFIX} \
               --min_tags=${MIN_TAGS}
+          ./pruneImages.sh \
```

Comment:
As a suggestion, if DCs and Jobs pruning commands are executed in different Github Actions steps, it will provide better visual feedback to the action execution.